src/GraphQL/Attributes/OutputTypeAttribute.cs

```diff
@@ -6,38 +6,22 @@ namespace GraphQL
     /// Specifies an output graph type mapping for the CLR class or property marked with this attribute.
     /// </summary>
     [AttributeUsage(AttributeTargets.Class | AttributeTargets.Interface | AttributeTargets.Struct | AttributeTargets.Property | AttributeTargets.Method | AttributeTargets.Field)]
-    public class OutputTypeAttribute : GraphQLAttribute
+    public class OutputTypeAttribute<TGraphType> : GraphQLAttribute
```

Comment:
Generic attributes save a couple lines here and provide better checks.

Reply:
> as it would require manually changing LangVersion (which is not always an acceptable answer). We have had issues before along the same lines with init;

I thought that adding LangVersion in csproj is not a big deal. Nevertheless it's only for demo purposes. I will try to change attributes to work in both cases.